Rubberduck.Interaction/IMessageBox.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Forms = System.Windows.Forms;
+
+namespace Rubberduck.Interaction
+{
+    public interface IMessageBox
+    {
+        void Notify(string text);
+        void NotifyError(string text, string caption);
+        void NotifyWarn(string text, string caption);
+        bool Question(string text, string caption);
+        bool Prompt(string text, string caption);
+        bool ConfirmYN(string text, string caption);
+        bool ConfirmYN(string text, string caption, bool suggestion);
+        bool? Confirm(string text, string caption, bool? suggestion);
+    }
+
+    public class MessageBox : IMessageBox
+    {
+        public void Notify(string text)
+        {
+            Forms.MessageBox.Show(text);
+        }
+
+        public void NotifyError(string text, string caption)
+        {
+            Forms.MessageBox.Show(text, caption, Forms.MessageBoxButtons.OK, Forms.MessageBoxIcon.Error);
+        }
+
+        public void NotifyWarn(string text, string caption)
+        {
+            Forms.MessageBox.Show(text, caption, Forms.MessageBoxButtons.OK, Forms.MessageBoxIcon.Exclamation);
+        }
+
+        public bool Question(string text, string caption)
+        {
+            return Forms.MessageBox.Show(text, caption, Forms.MessageBoxButtons.YesNo, Forms.MessageBoxIcon.Question) == Forms.DialogResult.Yes;
+        }
+
+        // FIXME check whether to merge with Question
+        public bool Prompt(string text, string caption)
```

Comment:
Yes definitely merge those into one. They do the same thing (NB: `Question` and `Prompt` likely came from two different uses of the old Windows enumerations, which is why we have 2 enumerations for same icons (c.f. [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms645505(v=vs.85).aspx) )